README.md

```diff
@@ -0,0 +1,71 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My First Capstone-Project (Mobile first)
+
+My portfolio project is prepared for different screen sizes. However, I built it by using "mobile first" approach. 
+
+I used this [template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
+
+
+## Built With
+
+- Library : Font Awesome
+- Framework : Boostrap
+- Technologies : Html & CSS & JavaScript
+
+
+## Live Demo
+
+[My First Capstone-Project](https://capstone-one.netlify.app/)
```

Comment:
- [ ] You have done a great job deploying your website using netlify server but it is required to deploy your project using GitHub pages so please change the deployment server.